cmd/validator/validator.go

```diff
@@ -133,6 +143,36 @@ func getFlags() (validatorConfig, error) {
 		}
 	}
 
+	uncheckedClean := cleanString("unchecked-files")
+	uncheckedPairs := strings.Split(uncheckedClean, ",")
+
+	// Check that checked files are listed in proper format and with valid config types
+	if uncheckedFiles != nil && isFlagSet("unchecked-files") {
+		for _, pair := range uncheckedPairs {
+			if !strings.Contains(pair, ":") {
+				errorMsg :="Format error in adding unchecked file: " + pair + ", ':' delimeter required to separate file path from type"
```

Comment:
I'm not convinced.

You could created an error with this

```go
err:= fmt.Errorf("Format error in adding unchecked file: %s, ':' delimeter required to separate file path from type", pair)
fmt.Println(err) 
flag.Usage() 
return validatorConfig{}, err
```

Reply:
Thanks for the review, been busy with another project for while.

I'm not quite sure what you meant in your feedback.

Is the error checking not robust enough, or is that a recommendation for better formatting the error output?

Reply:
It's about the code style, not really a formatting issue.

But usually when it's a kind of sentence and you are creating a new error, if you need to concatenate strings `fmt.Errorf(` is used